lm_eval/tasks/odex.py

```diff
@@ -0,0 +1,110 @@
+"""Execution-Based Evaluation for Open Domain Code Generation
+https://arxiv.org/pdf/2212.10481.pdf
+
+The ODEX dataset includes 945 NL-to-Code generation pairs with 1,707 
+human-written test cases. ODEX involves NL intents in four natural languages: 
+with 439, 90, 164, and 252 samples in English, Spanish, Japanese, and Russian.
+
+https://github.com/zorazrw/odex
+"""
+import os
+
+os.environ["HF_ALLOW_CODE_EVAL"] = "1"
```

Comment:
This is handled by the evaluation harness for tasks where `requires_execution=True`, to give users the option of not executing the code
```suggestion
```